apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -39,8 +39,14 @@ const TeacherHomepage = React.createClass({
         <ProtectedStatefulDiv
           ref="termsReminder"
         />
-        <Announcements
-          announcements={announcements}
+        <Notification
+          type="bullhorn"
+          notice={announcements[0].heading}
```

Comment:
I wonder if we'll want this to be more consistent language? For instance, pass in notices or notifications to the Notification instead of announcements.